### PR TITLE
feature(ComposerAlias) #21 Add scripts to composer.json for lint, php…

### PR DESCRIPTION
**Pull request by blanxii**
…cs, and phpunit

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -37,13 +38,18 @@
...
-        "lint": "parallel-lint src tests",
-        "phpcs": "phpcs src tests --standard=psr2 -sp --colors",
-        "phpunit": "phpunit --verbose --colors=always",
-        "test": [
-            "@lint",
-            "@phpunit",
-            "@phpcs"
-        ]
-    }
+       "lint": "parallel-lint src tests",
```
Comment (ramsey) [inline composer.json]: The indentation for these lines appears off. Please make sure it fits the indentation of the rest of the file (4 spaces per indentation level). Thanks!

**Comment by ramsey** [comment]
Are all the additional changes in the PHP files a result of running `phpcs` and `phpstan` on the code?

**Comment by blanxii** [comment]
@ramsey yes, I need to change some stuff to get phpstan make green

File: composer.json (hunk 1 above)
Comment (blanxii) [reply composer.json]: done!

**Comment by ramsey** [comment]
Thank you for contributing to this repository for #hacktoberfest! 🎉 

You check your progress here: https://hacktoberfest.digitalocean.com

**Pull request merged**